directory.json

```diff
@@ -161,5 +161,6 @@
 	"VOLGOGRAD": "https://raw.githubusercontent.com/ty221/fossasia-com/master/volgograd.json",
 	"X-LUG": "https://raw.githubusercontent.com/manastaneja/communities/master/Xiyou%20lug",
 	"YGNU": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Yemen/Linux%20Association",
-	"Yoko-lug" : "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Japan/YokohamaLinuxUserGroup-api.json"
-}
\ No newline at end of file
+	"Yoko-lug" : "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Japan/YokohamaLinuxUserGroup-api.json",
+	"YSITD":"https://raw.githubusercontent.com/Sea-n/fossasia-communities/master/Taiwan/YSITD.json"
+}
```

Comment:
Hey, Shouldn't the Community name go up there instead of your name i.e. in my opinion it should be `"Fixis": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Taiwan/YSITD.json"` ?